### PR TITLE
Add Protocol 22 to Software Versions

### PR DESCRIPTION
**Pull request by briwylde08**

### CODE REVIEW

File: docs/networks/software-versions.mdx [hunk 1]
```diff
@@ -12,6 +12,42 @@ Release candidates are software releases that are also released to the [Testnet]
...
+| Soroban Rust SDK | `21.0.1-preview.3` |
+| Stellar CLI | `v21.0.0` |
+| Soroban RPC | `v21.3.0` |
+| Stellar Horizon | `v2.30.0` |
+| Stellar Quickstart | `https://hub.docker.com/layers/stellar/quickstart/v426-latest-amd64/images/sha256-274395daab6fa8033b9213f152d56699358917fb01d7c7e95392a37fc00c9d01` |
+| Stellar JS Stellar Base | `v12.0.0-rc1` |
+| Stellar JS Stellar SDK | `v12.1.0` |
+| Freighter |  |
+| Laboratory |  |
+| Soroban React Payment dapp | `TBD` |
```
Comment (anupsdf) [inline docs/networks/software-versions.mdx:34]: I wonder if we should remove these 3 dapp entries. They have been in TBD for quite some time.

Comment (briwylde08) [reply docs/networks/software-versions.mdx]: Works for me.

File: docs/networks/software-versions.mdx [hunk 2]
```diff
@@ -12,6 +12,47 @@ Release candidates are software releases that are also released to the [Testnet]
...
+### Software
+
+| Software | Version |
+| --- | --- |
+| XDR | `v22.0` |
+| Rust XDR | `v22.0.0-rc.1.1` |
+| Soroban Environment | `v22.0.0-rc.2` |
+| Stellar Core | `v22.0.0rc2` |
+| Soroban Rust SDK | `v22.0.0-rc.2.1` |
+| Stellar CLI | `v21.0.0` |
```
Comment (anupsdf) [inline docs/networks/software-versions.mdx:26]: How about leaving the downstream v21 versions as TBD for now? As the teams release their versions they can update this.

Comment (sisuresh) [reply docs/networks/software-versions.mdx]: Good point. Done

**Review by anupsdf — APPROVED** [review]

**Pull request merged**